Bottom: 6574bc1c7d3ee0a6b4141c942afcb2d7e82e4844
Top:    e09be88b5664d78185e91421d69c423f28403661
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-27 20:24:11 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
